cmd/login.go

```diff
@@ -25,6 +26,7 @@ import (
 
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
+    log.Information("okteto login will soon be depreciated, run okteto context to configure your cluster.")
```

Comment:
I would use `log.Warning`. It will set the color to yellow which is the standard for warnings

Reply:
Also, the indentation doesn't seem right, at least, I see it a bit to the left

Reply:
We also surround okteto commands with '' so the user knows where the command starts and ends. 
`okteto login will soon be depreciated, run okteto context to configure your cluster.` -> `'okteto login' will soon be depreciated, run 'okteto context' to configure your cluster.`

Reply:
This is what I mean, it seems `log....` is not aligned with the next line, at least this is how I see it in GitHub, not sure if it is right in the code or not

![Captura de pantalla 2021-11-04 a las 11 26 50](https://user-images.githubusercontent.com/3510171/140298201-9ffb4692-6948-4604-9081-f7eb71d270f7.png)

Reply:
Yep. I thought of this. I'll change this.

Reply:
Oh @ifbyol, interestingly, it's on the same line in my IDE 😬